File: app/src/main/java/com/mgaetan89/showsrage/model/RootDir.java
package com.mgaetan89.showsrage.model;

import com.google.gson.annotations.SerializedName;

public class RootDir {
	@SerializedName("default")
	private int defaultDir = 0;
	private String location = "";
	private int valid = 0;

	public int getDefaultDir() {
		return this.defaultDir;
	}

	public String getLocation() {
		return this.location;
	}

	public int getValid() {
		return this.valid;
	}
}

File: app/src/main/kotlin/com/mgaetan89/showsrage/model/RootDir.kt
package com.mgaetan89.showsrage.model

import com.google.gson.annotations.SerializedName
import io.realm.RealmObject
import io.realm.annotations.PrimaryKey

open class RootDir : RealmObject() {
    @SerializedName("default")
    open var defaultDir: Int = 0
    @PrimaryKey
    open var location: String = ""
    open var valid: Int = 0
}
